java/source/org/libreoffice/ide/eclipse/java/export/JavaExportPart.java

```diff
@@ -66,9 +63,20 @@
     private Button mSaveScripts;
     private Composite mNameRow;
     private Label mNameRowLbl;
-    private Text mNameRowTxt;
+    private Label mNameRowValueLbl;
 
     private JavaExportPageControl mController;
+    private static AntScriptExportWizardPage mAntScriptPage;
+
+    public static final int HORIZONTAL_INDENT = 20;
+
+    /**
+     * @param pAntScriptPage
+     *            Helps in retrieving the project selected in this part.
+     */
+    public static void setAntScriptExportPage(AntScriptExportWizardPage pAntScriptPage) {
```

Comment:
Please no references to the ant stuff from here. If you need a shared method, add it to some helper class or something.